### ISSUE TITLE
deck.gl ES5 and ES6 dist files are exactly identical

### ISSUE THREAD

**Issue by LINKIWI** [uber/deck.gl#1805]
I don't believe this is expected behavior: the ES5, ES6, and ESNext dist files are all exactly identical after installing from npm.

Some basic reproduction steps:

```bash
$ yarn add deck.gl  # or npm install deck.gl
$ cd node_modules/deck.gl
$ diff dist/es5/index.js dist/es6/index.js  # no differences
```

This leaves stray `export` statements in the ES5 dist file. This causes many Node runtimes to error since project-level defined transformers usually ignore files in `node_modules`.

```js
export { // CONSTANTS
COORDINATE_SYSTEM, // Main class
Deck, // Base Layers
Layer, CompositeLayer, // Views
View, MapView, FirstPersonView, ThirdPersonView, OrbitView, PerspectiveView, OrthographicView, // Viewports
Viewport, WebMercatorViewport, PerspectiveViewport, OrthographicViewport, // Controllers
MapController, // For custom layers
AttributeManager, // Shader modules
project, project64, lighting, // Internal classes
LayerManager } from '@deck.gl/core'; // EXPERIMENTAL CORE LIB CLASSES (May change in minor version bumps, use at your own risk)
```

The user-facing error here is that this is causing errors in a `jest` testing environment, for [example](https://travis-ci.org/LINKIWI/orion-web/builds/378196429):

```
 FAIL  test/index.test.js
  ● Entry › Rendering into DOM node

    /home/kiwi/sync/code/orion-web/node_modules/deck.gl/dist/es5/index.js:26
    export { // CONSTANTS
    ^^^^^^
    
    SyntaxError: Unexpected token export

      19 |     <div style={{ bottom: 0, position: 'absolute', right: 0, zIndex: 2 }}>
      20 |       <Spacing right>
    > 21 |         <Spacing size="large" bottom>
      22 |           <NavigationControl onViewportChange={onViewportChange} />
      23 |         </Spacing>
      24 |       </Spacing>
      
      at ScriptTransformer._transformAndBuildScript (node_modules/jest-runtime/build/script_transformer.js:316:17)
      at Object.<anonymous> (src/app/react/components/map/index.js:21:13)
      at Object.<anonymous> (src/app/react/containers/map.js:33:13)

```

**Comment by LINKIWI** [comment]
For example, I found this after some Googling: https://stackoverflow.com/a/49651756

The proposed solution feels like a workaround. Shouldn't the ES5 dist file already be transpiled?

**Comment by Pessimistress** [comment]
Thanks for reporting. This seems to be broken only in the last patch:
https://unpkg.com/deck.gl@5.2.4/dist/es5/index.js
vs
https://unpkg.com/deck.gl@5.2.3/dist/es5/index.js

You may want to downgrade as we investigate.

**Comment by jianhuang01** [comment]
@LINKIWI published 5.2.5, the issue should be fixed.

**Comment by LINKIWI** [comment]
5.2.5 seems to fix the issue, thanks!

There is now a separate error emitted from luma.gl related to using WebGL in a Node environment. I'll open a separate issue for that.
